distributed/tests/test_stress.py

```diff
@@ -229,32 +229,32 @@ async def test_stress_steal(c, s, *workers):
             break
 
 
-@pytest.mark.slow
-@gen_cluster(
-    nthreads=[("", 1)] * 10,
-    client=True,
-    timeout=180,
-    scheduler_kwargs={"transition_counter_max": 500_000},
-    worker_kwargs={"transition_counter_max": 500_000},
-)
+# @pytest.mark.slow
+@pytest.mark.repeat(50)
+@gen_cluster(client=True, nthreads=[("", 1)] * 10)
 async def test_close_connections(c, s, *workers):
+    # Schedule 600 slowinc's interleaved by worker-to-worker data transfers
+    # The minimum time to compute this is (600 * 0.1 / 10 threads) = 6s
     da = pytest.importorskip("dask.array")
-    x = da.random.random(size=(1000, 1000), chunks=(1000, 1))
+    x = da.random.random(size=(100, 100), chunks=(-1, 1))
     for _ in range(3):
-        x = x.rechunk((1, 1000))
-        x = x.rechunk((1000, 1))
-
-    future = c.compute(x.sum())
```

Comment:
This graph is 15_000 tasks that individually release the GIL for microseconds. Yikes.